internal/lib/lib.go

```diff
@@ -67,14 +67,29 @@ func SetDisableSync(state bool) {
 var AKOUser string
 
 func SetAKOUser() {
-	AKOUser = "ako-" + GetClusterName()
+	//Set AKO user same as login user if GRBAC enabled
+	if GetEnableGRBAC() {
+		AKOUser = os.Getenv("CTRL_USERNAME")
```

Comment:
By GRBAC with AKO design, customer has to make sure to use per cluster unique user. Objects from each cluster will be tagged with different labels. If user is duplicated across two cluster, until and unless that user is associated with labeled role which corresponds to clusters, it will not able to view or do REST operations on those objects at AVI Portal side.